packages/curve-ui-kit/src/hooks/useTabFromQueryString.ts

```diff
@@ -0,0 +1,49 @@
+import { useCallback } from 'react'
+import { useNavigate, useLocation } from 'react-router-dom'
+
+type Tab = { value: string; label: string }
```

Comment:
We could add the component here as well to avoid [repeating all the options](https://github.com/curvefi/curve-frontend/blob/795e0f5163eede1ba0590587498a1333ed0784d2/packages/curve-ui-kit/src/widgets/Disclaimer/Disclaimer.tsx#L47-L50)
```suggestion
type Tab = { value: string; label: string, render: FunctionComponent }
```

Reply:
That's what I had on the first attempt, but then I realized the TabSwitcher itself doesn't render the component, and having to find the active tab with `.find()` was more cumbersome than just lines of easy code 😅